tests/test_erisx.py

```diff
@@ -49,11 +49,12 @@ def setup_method(self):
         orig_get_account = ErisxApi.get_account
         FixEngine.logon = do_nothing
         ErisxApi.get_account = do_nothing
+        self.account_id = 0
         self.client = ErisxApi(fix_trading_endpoint="127.0.0.1:1752", fix_trading_user="test",
                                fix_marketdata_endpoint="127.0.0.1:1753", fix_marketdata_user="test",
                                password="test",
                                clearing_url="https://127.0.0.1/api/v1/",
-                               api_key="key", api_secret="secret", account_id=0)
+                               api_key="key", api_secret="secret", certs={}, account_id=self.account_id)
```

Comment:
Is the behavior passing an empty dictionary the same as passing `None`?

Reply:
Functionally yes, but would cause issues if FIX tests were ever added. Replaced with `None`